### ISSUE TITLE
Separate each line of the quotes and add spacing between each paragraph

### ISSUE THREAD

**Issue by xidream** [whateverzpy/LeetCode-Markdown#1]
It would look great to me if we could separate each line of the quotes and add spacing between each paragraph!

To separate each line of the quotes, we could add `+ " <br>"` after the `line` here:
https://github.com/whateverzpy/LeetCode-Markdown/blob/4ebf864fcc72b99093fea540bc7c224fbd3ca803/content-scripts/content.js#L23

To add spacing between each paragraph, we could probably check if the `content` is `"Example 1:"`, `"Constraints:"`, or `"Follow\-up:"`, and add `"<br>\n\n"` before `"**"` for it:
https://github.com/whateverzpy/LeetCode-Markdown/blob/4ebf864fcc72b99093fea540bc7c224fbd3ca803/content-scripts/content.js#L8-L10

**Comment by whateverzpy** [comment]
Can you come up with a specific example of a leetcode problem and show how it works? Thanks!

**Comment by xidream** [comment]
Sure! :)

So [here](https://github.com/xidream/leetcode/blob/889e2dda17d3d3284da10a6f57076cde6e9e90ff/problem/0003_longest_substring_without_repeating_characters/README.md) is a markdown file generated from [3. Longest Substring Without Repeating Characters](https://leetcode.com/problems/longest-substring-without-repeating-characters/). 
^ As you can see, there are no line breaks between **Input**, **Output**, **Explanation** in the **Example 1**, **2**, **3** in the Preview.

And [here](https://github.com/xidream/leetcode/blob/3d60b39906ae1579d8c4dbf154f8218d615f81fd/problem/0003_longest_substring_without_repeating_characters/README.md) is a version I'm looking forward to. 
^ In addition to adding line breaks into the Examples, we may also add a little spacing between the description, Examples, and the Constraints!

[Here](https://github.com/xidream/leetcode/commit/3d60b39906ae1579d8c4dbf154f8218d615f81fd?diff=unified) is the diff.
^ As you can see, I simply add `<br>` tags at the end of each line of the Examples, and the places I would like to add some spacing.

But actually, I did notice that this line break thing is kind of a markdown parser issue after I posted this issue...

Like you would also need two line breaks to be able to move the text after the cursor to a different line in a markdown file on GitHub, or it will still in the same line. But in here, the comment block I'm writing right now, one line break moves the text after the cursor to a new line. You don't need additional `<br>` tags to separate each line of the Examples if you paste the generated markdown here...

So I'm thinking about adding some options (checkboxes) into it, something like 

- [ ] with `<br>` tags at the end of each line in a quote.
- [ ] with `<br>` tags for the spacing between each section.

Then users can choose the markdown format they want. 

But not sure if this would be a feature you would like to add...

**Comment by whateverzpy** [comment]
Whether in an online Markdown editor, Typora, or VS Code, the `input`, `output`, and `explanation` sections seem to be rendered consistently with the original problem format. So I decided not to add this feature.
![image](https://github.com/user-attachments/assets/31897249-56ba-4a39-abaa-9e2565306d5a)
![image](https://github.com/user-attachments/assets/f56e1146-1332-483b-a170-20c077285780)
However, the unexpected line break occurred in the `longest substring` section above.(I will try to fix it, maybe you can help 😂)
Anyway, thanks!

**Comment by xidream** [comment]
Yeah, I did notice some editors deal with the Markdown syntax as we expect. 

(My VS Code does not though😖...)

![image](https://github.com/user-attachments/assets/046a060b-62c2-42c1-af0e-0078fbe8cf8c)

However, I saw [this answer](https://stackoverflow.com/a/33191810/19959651) on Stack Overflow. It includes the original Markdown specification. 

You can see that the creator of Markdown intended not to convert every line break to a `<br/>` tag. 

But what really caught my attention was:
> When you do want to insert a `<br />` break tag using Markdown, you end a line with two or more spaces, then type return.

I would probably never know this if I didn't see it here😂. 

So, instead of adding `<br>` tags directly into the Markdown, what we should do (if we want) is to add two spaces at the end of the line. Then the next line will become a new line in standard Markdown syntax!

I believe adding two spaces at the end of the line won't even cause an additional line break into those editors you mentioned? What do you think, would you like to do it if so?

As for adding spacing between each section, I see there are actually `<p>&nbsp;</p>` tags there. Would you like to try to convert them to `<br>` tags? (Two spaces does not seem to work for adding the spacing...)

Yeah, I did notice the "substring" issue there haha. But I couldn't find out the reason easily. Actually, I'm not even very familiar with JavaScript or HTML, or have an environment to test and fix it. I just read your code and guess actually 🤣. I could create a new issue for this though... (if you want)🤣🤣🤣

**Comment by whateverzpy** [comment]
Please email me. 2023311810@stu.hit.edu.cn Perhaps we can exchange more information.
